integration-tests/test/basic-l1-l2-communication.spec.ts

```diff
@@ -97,6 +96,25 @@ describe('Basic L1<>L2 Communication', async () => {
       expect((await L2SimpleStorage.totalCount()).toNumber()).to.equal(1)
     })
 
+    it('should be able to send L1 -> L2 directly via enqueue', async () => {
```

Comment:
This name should be a bit more descriptive that it covers the fact `CALLER` is the default value so its easier to remember that is covered via tests